client/src/Views/Home/Prospects/ProspectSidebar.js

```diff
@@ -0,0 +1,103 @@
+        height: "100vh",
+        width: "100%",
+    },
+    textField: {
+        width: '60%',
+        marginLeft: 'auto',
+        marginRight: 'auto',            
+        paddingBottom: 0,
+        marginTop: 50,
+        fontWeight: 500,
+    },
+  }));
+
+function ProspectSidebar (props) {
+
+    const [state, setState] = React.useState({
+        limitCheckbox : false,
+        tadCheckbox : false,
+        importedFromCheckbox : false,
+        optedOutCheckbox : false
+      });
+
+    const handleChange = (event) => {
+    setState({ ...state, [event.target.name]: event.target.checked });
+    };
+
+    const handleInputChange = (event) => {
+        const text = event.target.value;
+        props.onChange(props.id, text);
+    }
+
+    const classes = useStyles();
+    return (
+        <Box className={classes.main}>
+            <Grid
+                className={classes.root}
+                container
+                direction="column"
+                alignItems="center"
+                wrap="nowrap"
+            >
+                <TextField 
+                className={classes.textField}
+                onChange={handleInputChange} value={props.value}
+                id="outlined-basic" label="Search" variant="outlined"
+                InputProps={{
+                    startAdornment: (
+                        <InputAdornment position="start">
+                        <SearchIcon style={{ color: "#4FBE75" }}/>
+                        </InputAdornment>
+                    ),
+                    }}
+                /><br/>
+                <FormControl component="fieldset" className={classes.formControl}>
+                    <FormGroup>
+                    <FormControlLabel
+                        control={<SidebarCheckbox disableRipple={true} checked={state.importedFromCheckbox} onChange={handleChange} name="importedFromCheckbox" />}
+                        label="Imported From"
+                    />
+                    <FormControlLabel
+                        control={<SidebarCheckbox disableRipple={true} checked={state.optedOutCheckbox} onChange={handleChange} name="optedOutCheckbox" />}
+                        label="Opted Out"
+                    />
+                    <FormControlLabel
+                        control={<SidebarCheckbox disableRipple={true} checked={state.bouncedCheckbox} onChange={handleChange} name="bouncedCheckbox" />}
+                        label="Bounced"
+                    />
+                    <FormControlLabel
+                        control={<SidebarCheckbox disableRipple={true} checked={state.tadCheckbox} onChange={handleChange} name="tadCheckbox" />}
+                        label="Tad"
+                    />
+                    <FormControlLabel
+                        control={<SidebarCheckbox disableRipple={true} checked={state.limitCheckbox} onChange={handleChange} name="limitCheckbox" />}
+                        label="Limit"
+                    />
```

Comment:
lots of repeated code here.. but I understand taht this doesn't do anything at the moment, so no point changing - however I'd prefer using a map here in the future